Firestore/core/test/firebase/firestore/testutil/testutil.h

```diff
@@ -40,6 +46,24 @@ inline model::ResourcePath Resource(absl::string_view field) {
   return model::ResourcePath::FromString(field);
 }
 
+// Version is epoch time in microseconds. This helper is defined so to match
+// SDKs in other platform.
+inline model::SnapshotVersion Version(int64_t version) {
+  return model::SnapshotVersion{
+      Timestamp{/*seconds=*/version / 1000,
+                /*nanoseconds=*/static_cast<int32_t>(version % 1000) * 1000}};
```

Comment:
Similarly, should this be `(version % (1000 * 1000)) * 1000`? Or is `version` in milliseconds?